.travis.yml

```diff
@@ -51,6 +52,7 @@ install:
 
 script:
   - if [[ $COVERAGE = 1 ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
+  - if [[ $STATIC_ANALYSIS = 1 ]]; then composer require nunomaduro/larastan && composer stan; fi
```

Comment:
So this means the check will only be done by Travis

Reply:
At the moment, yes.